RawWaterManagement/SluiceGate/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,142 @@
+{
+  "location": {
+    "type": "Property",
+    "value": {
+      "type": "Point",
+      "coordinates": [
+        -26.8150565,
+        -128.911332
+      ]
+    }
+  },
+  "address": {
+    "type": "Property",
+    "value": {
+      "streetAddress": "Its for four. Order pass rich wide material deal. Group dog property painting.",
+      "addressLocality": "Culture condition involve. North us really door newspaper. Door any again.",
+      "addressRegion": "Audience watch much measure. Admit we across know fill since.",
+      "addressCountry": "Her million artist management almost.",
+      "postalCode": "Land do avoid watch next economic today. Perhaps practice money. Measure rather no identify happen plan.",
+      "postOfficeBoxNumber": "Today from indeed measure. Father since child. This information difference program policy full. Foreign contain live.",
+      "areaServed": "Away defense politics. Parent scientist source town federal crime. All short drive while south skin."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Court item business when idea thousand anything. Something know option as. Begin operation show place attack lead itself."
+  },
+  "id": "urn:ngsi-ld:dataModel:id:EGUP:16321644",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2003-06-19T06:37:36Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2019-02-20T21:54:19Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Else standard campaign never."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Military successful report wear necessary let research. Decade choice cell."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Cost tell site concern cost. West whom hope feel nothing candidate political. Government knowledge dog inside."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Too commercial a step. Bill another choice next suffer. Generation bank spend happen police."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Than produce spend executive talk fall relationship. Place fire require each up yet participant."
+  },
+  "owner": {
```

Comment:
Type suggested toi be relationship
1. Relationships should not be modelled as properties, otherwise you completely lost the interest of NGSI-LD which is based on graph. 
2. Arrays in NGSI-LD are stored as strings so you can not use the API to query indiviudual alements in a array. 
3. Relationships can not be provided within arrays.

When you need to have several relationships with the same name, then you have to use the datasetId 

Example for a relationship where one entity would "monitor" 2 different rooms. It thus have 2 relationshipsnames "monitored" directed toward room01 and room02. The datasetId allows to differentiate the 2 relationships.
"monitors":[
	{
	   "type":"Relationship",
	   "object":"urn:ngsi-ld:Room:01",
	   "datasetId":"urn:ngsi-ld:dataSet:01"},
	},
	"type":"Relationship",
	"object":"urn:ngsi-ld:Room:02" ,
	"datasetId":"urn:ngsi-ld:dataSet:02"}
	]